src/net.h

```diff
@@ -233,8 +233,8 @@ struct LocalServiceInfo {
 extern Mutex g_maplocalhost_mutex;
 extern std::map<CNetAddr, LocalServiceInfo> mapLocalHost GUARDED_BY(g_maplocalhost_mutex);
 
-extern const std::string NET_MESSAGE_COMMAND_OTHER;
-typedef std::map<std::string, uint64_t> mapMsgCmdSize; //command, total bytes
+extern const std::string NET_MESSAGE_OTHER;
+typedef std::map<std::string, uint64_t> mapMsgSize; //command, total bytes
```

Comment:
Shouldn't `mapMsgCmdSize` be renamed to `mapMsgTypeSize` ?

Reply:
I am going to disagree - mapMsgSize seems correct (IMO)